scripts/update.py

```diff
@@ -288,6 +299,101 @@ def format_df_for_chirp(df: pd.DataFrame) -> pd.DataFrame:
 
     return df
 
+def format_df_for_d878(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Format a DataFrame of repeaters for use in Anytone D878.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        All of the repeaters.
+
+    Returns
+    -------
+    pd.DataFrame
+        The FM and NBFM repeaters, and DMR repeaters in the 2m and 70cm bands,
+        formatted for Anytone D878.
+    """
+
+    print("Generating Anytone D878 CSV file.")
+    total_repeaters = len(df.index)
+
+    # Select FM and DMR channels.
+    df = df.loc[df["Mode"].isin(["FM", "NBFM", "DMR"])].copy()
```

Comment:
In the generic file, we include the Fusion repeaters because they can take standard analog FM if you aren't using Fusion mode. I think we should include those here in the same manner, treating them as FM, no ?

Reply:
Oh!  I didn't know that.  I thought Fusion used a digital mode?

    utilizing C4FM 4-level FSK Technology to transmit digital voice and data over the Amateur radio

Reply:
Oh - I see there is a compatability mode:

    Analog FM Mode

    Maintains backwards compatibility with existing Analog FM Equipment, allowing a wide range of users to experiment 
    with System Fusion Digital.

Reply:
Yes -- I confirmed that most Fusion repeaters fuse ( hihi ) analog of Yaesu digital magic, so I assumed people would prefer having them as FM than not at all !